core/src/main/java/com/linecorp/armeria/client/Clients.java

```diff
@@ -30,6 +30,7 @@
 
 /**
  * Creates a new client that connects to a specified {@link URI}.
+ * If you want to make an {@link HttpClient}, using the factory methods in {@link HttpClient} is recommended.
```

Comment:
How about: `If you are creating an {@link HttpClient}, it is recommended to use the factory methods in. ..`